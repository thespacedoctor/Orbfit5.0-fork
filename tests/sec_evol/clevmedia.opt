50      50      0.01     90        0.00001
ngre    ngro    emin     ommax     eps
