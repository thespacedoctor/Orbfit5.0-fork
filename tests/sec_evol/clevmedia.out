2007VA85.out